AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_PhysicalSkyAdded.py

```diff
@@ -71,23 +122,34 @@ def AtomEditorComponents_PhysicalSky_AddedToEntity():
     2) Add Physical Sky component to Physical Sky entity.
     3) UNDO the entity creation and component addition.
     4) REDO the entity creation and component addition.
-    5) Update Sky Intensity value.
-    6) Enter/Exit game mode.
-    7) Test IsHidden.
-    8) Test IsVisible.
-    9) Delete Physical Sky entity.
-    10) UNDO deletion.
-    11) REDO deletion.
-    12) Look for errors and asserts.
+    5) Intensity mode set to Nit then back to Ev100.
+    6) Set Sky Intensity value to 0 then back to 4.
+    7) Set Sun Intensity value to 0 then back to 8.
+    8) Set Turbidity value to 5 then back to 1.
+    9) Set Sun Radius Factor from 0.0 back to 1.0.
+    10a) Enable Fog for subsequent tests.
+    11) Change fog color to green, then return to default.
+    12) Set Fog Top Height to 0.5 then back to 0.01.
+    13) Set Fog Bottom Height to 0.3 then back to 0.0.
+    10b) Disable Enable Fog to set back to default.
+    14) Enter/Exit game mode.
+    15) Test IsHidden.
+    16) Test IsVisible.
+    17) Delete Physical Sky entity.
+    18) UNDO deletion.
+    19) REDO deletion.
+    20) Look for errors and asserts.
 
     :return: None
     """
 
     import azlmbr.legacy.general as general
+    import azlmbr.math as math
```

Comment:
depending on how much you are using from azlmbr.math you might remove this import and add explicit classes to the line below
from azlmbr.math import Math_IsClose, Vector3

Reply:
Fixed